src/shared/ACSEngine.test.ts

```diff
@@ -0,0 +1,7 @@
+import { ACSEngine } from "./ACSEngine";
+
+describe("ACSEngine", () => {
+  it("Should return getACSEnginePath", () => {
+    expect(ACSEngine.getACSEnginePath()).toContain(ACSEngine.acsProjectPath);
```

Comment:
Might look into changing the behavior of getACSEnginePath in the future, but this is good for now